src/wallet/wallet.h

```diff
@@ -63,6 +64,7 @@ static const bool DEFAULT_WALLET_REJECT_LONG_CHAINS = false;
 static const unsigned int DEFAULT_TX_CONFIRM_TARGET = 6;
 //! -walletrbf default
 static const bool DEFAULT_WALLET_RBF = false;
+static const bool DEFAULT_WALLET_QT_RBF = true;
```

Comment:
Eh. I guess we don't want Qt code to end up here

Reply:
I'm not sure how to disentangle this. The `-walletrbf` is cast to a boolean in init.cpp, so QT can't tell if that flag was present or not.

Reply:
What about a separate option for the GUI, GUI-only, that defaults to `-walletrbf`? This would allow it to be completely disentangled from the core.